### PR TITLE
Update Text in Upload Dataset View

### LINKED ISSUE #7760: Update the text in the "Upload Dataset" Screen in the Open Source version
## Detailed Description
Change the text displayed on the "Upload Dataset" screen to the following:
"Drag and drop your files in WKW format.
Need to upload files in other formats? Switch to [webknossos.org](http://webknossos.org/) for more file types support and automatic conversion.
[Learn more!](https://webknossos.org/self-hosted-upgrade)"

Screenshot 2024-04-15 at 14 23 51

## Context
We want to encourage users of the open-source version to consider migrating to webknossos.org. The "Upload Dataset" screen, which currently accepts only WKW format, is an ideal place to inform users about the extended capabilities available on webknossos.org. Highlighting the support for more file formats on our full platform can motivate users to explore and potentially switch.

### PR DESCRIPTION
**Pull request by dieknolle3333**
https://updateuploaddstext.webknossos.xyz/

### Steps to test:
- Click "upload dataset"
- Toggle the options `recommendWkorgInstance` and `isDatasetConversionDisabled` to check that all combinations look nice
- Check links

### To do
- [x] fix tests: update snapshot misc.e2e.js.snap

### Issues:
- fixes #7760

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: CHANGELOG.unreleased.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 - Because of the way our models are trained, AI analysis and training is disabled for 2D and ND datasets, as well as for color layers with data type uInt24. [#7957](https://github.com/scalableminds/webknossos/pull/7957)
 - The overall performance was improved (especially for the segments tab). [#7958](https://github.com/scalableminds/webknossos/pull/7958)
 - The performance for the skeleton tab was improved. [#7989](https://github.com/scalableminds/webknossos/pull/7989)
+- For self-hosted versions, the text in the data set upload view was updated to recommend switching to webknossos.org. [#7996](https://github.com/scalableminds/webknossos/pull/7996)
```
Comment (philippotto) [inline CHANGELOG.unreleased.md:41]: please also update the migration guide (`MIGRATIONS.unreleased.md`) with a hint for that new config flag in application.conf.

**Review by philippotto — COMMENTED** [review]
great! only see my one comment.

**Comment by dieknolle3333** [comment]
there were merge conflicts in `misc.e2e.js.snap` so I committed the refreshed snapshots within the latest merge commit. I hope this is correct.

**Comment by dieknolle3333** [comment]
some screenshots :) 
image
image
image

**Review by philippotto — APPROVED** [review]
great :+1: thank you!

**Comment by dieknolle3333** [comment]
I just realized that this is buggy. For me clicking the links does not work properly. I will fix this :sweat_smile:

**Comment by dieknolle3333** [comment]
One link was relative. Next to that, clicking the links opens the file browser, as does clicking anywhere in the upload view. That doesnt feel convenient, so I will stop the event propagation on the links I think

**Pull request merged**